contracts/asset-proxy/contracts/src/interfaces/IAuthorizable.sol

```diff
@@ -18,7 +18,7 @@
 
 pragma solidity ^0.5.9;
 
-import "@0x/contracts-utils/contracts/src/interfaces/IOwnable.sol";
+import "../../archive/interfaces/IOwnable.sol";
```

Comment:
Hmm.

Reply:
There probably wasn't a reason to archive this in retrospect. The reason I did was because the Solidity version is different, but I'll go ahead and remove.